robot/robot.go

```diff
@@ -158,6 +158,9 @@ type Robot interface {
 
 	// Version returns version information about the robot.
 	Version(ctx context.Context) (VersionResponse, error)
+
+	// TrafficTunnelEndpoints returns information on available traffic tunnels.
+	TrafficTunnelEndpoints() []config.TrafficTunnelEndpoint
```

Comment:
I don't love adding to/breaking this `Robot` interface, but I feel like this is a case where we'll need to. Especially when I add `ListTunnels` in a forthcoming PR, the `ListTunnels` method in `robot/server.go` will need access to the reported, available tunnels on the actual robot.

Reply:
would not be opposed to adding a ListTunnels method on the Robot interface that also returns the timeout (I think it's fine to add to api as well) to keep things in sync instead of adding 2 very similar methods - or put TrafficTunnelEndpoints in the LocalRobot interface and do a cast inside Tunnel(). I

Reply:
Good point; I think I'll rename this method to `ListTunnels` and plan on that eventual API also returning the `connection_timeout` for the tunnel.